reference/shipping_provider.yml

```diff
@@ -1940,30 +1882,21 @@ components:
                       description: The value associated with the meta field
                     namespace:
                       type: string
-                      description: The namespace associated with
-                        product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield)
-                        or product variant
-                        (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield)
-                        meta fields. The meta field namespace should be saved
-                        under this format `shipping_carrier_{yourCarrierId}`
-                        otherwise it will not be recognized as an attribute.
+                      description: 'The namespace associated with product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) or product variant (/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield) meta fields. The meta field namespace should be saved under this format `shipping_carrier_{yourCarrierId}` otherwise it will not be recognized as an attribute.'
```

Comment:
```suggestion
                      description: 'The namespace associated with a product(/api-reference/catalog/catalog-api/product-metafields/createproductmetafield) or product variant(/api-reference/store-management/catalog/product-variants-metafields/createvariantmetafield) meta fields. You should save a meta field namespace under this format `shipping_carrier_{yourCarrierId}`; otherwise, you will not be able  to recognize it as an attribute.'
```